WORKFLOW.md

```diff
@@ -53,6 +53,10 @@ You can read about Rubocop [here](http://rubocop.readthedocs.io/en/latest/).
 Make a fork of the repository, create a new branch for every feature you are
 working on! **Always make pull requests!**
 
+### HOW TO: Pull Requests & Syncing
+
+You can learn more about how to create a pull request and sync your forked repo with the upstream(ifme) repo [here](https://github.com/julianguyen/ifme/wiki/HOW-TO:-Pull-Requests-&-Syncing)
```

Comment:
Can you add a space between `upstream(ifme)` i.e. `upstream (ifme)`?

Reply:
@julianguyen sure I can make the changes. But, I wanted to know, do you want the `ifme` in parenthesis or not? i.e do you want `upstream ifme` or `upstream (ifme)`. Also, should I remove the words `HOW TO:` from the title of the [wiki](https://github.com/julianguyen/ifme/wiki/HOW-TO:-Pull-Requests-&-Syncing) as well or just from `WORKFLOW.md`. let me know and I will happily make the changes.

Reply:
`upstream (ifme)` should be fine!

Yeah let's remove `HOW TO:` from both areas :)

Thanks a bunch!